packages/concerto-util/lib/basefileexception.js

```diff
@@ -31,9 +32,10 @@ class BaseFileException extends BaseException {
      * @param {string} fullMessage - the optional full message text
      * @param {string} [fileName] - the file name
      * @param {string} [component] - the component which throws this error
+     * @param {string} errorType - The optional error code regarding the error
```

Comment:
```suggestion
     * @param {string} [errorType='BaseFileException'] - the error code
```
[Docs](https://jsdoc.app/tags-param#:~:text=An%20optional%20parameter%20and%20default%20value).